xbmc/input/actions/Action.h

```diff
@@ -91,7 +93,7 @@ class CAction
   bool IsAnalog() const;
 
 private:
-  int          m_id;
+  int m_id = ACTION_NONE;
```

Comment:
Does default init justify the additional include needed now? IMO not.

Reply:
removed this change